packages/sdk/src/oracle/common/types/interfaces.ts

```diff
@@ -0,0 +1,177 @@
+import { TransactionConfirmer } from "../utils";
```

Comment:
this file defines interfaces that should be the same across clients using different oracles